public/app/features/variables/query/QueryVariableEditor.tsx

```diff
@@ -130,8 +131,7 @@ export class QueryVariableEditorUnConnected extends PureComponent<Props, State>
       return null;
     }
 
-    const datasource = extended.dataSource;
-    const VariableQueryEditor = extended.VariableQueryEditor;
+    const datasource: DataSourceRef = { uid: variable.datasource?.uid, type: variable.datasource?.type };
```

Comment:
is variable.datasource not already a DataSourceRef?

Reply:
Ups, you're right, I was getting it before from extended, which is a DataSourceAPI, and I left it like that, good catch Torkel!